Detectors/EMCAL/workflow/src/EMCALDigitizerSpec.cxx

```diff
@@ -94,6 +98,81 @@ void DigitizerSpec::run(framework::ProcessingContext& ctx)
   TStopwatch timer;
   timer.Start();
 
+
+
+
+
+
+  auto& eventParts = context->getEventParts();
+
+
+  // ------------------------------
+  // TRIGGER Simulation
+  // ------------------------------
+  // 1. Run SDigitizer separate loop -> map of vector of SDigits
+  // 2. Run Trigger simulation chain (Digitizer -> Digits writeout buffer -> L0 Simulation)
+  // 3. Run Loop SDigits -> Digits, set live only if the trigger is accepted.
+  //
+  // loop over all composite collisions given from context
+  // (aka loop over all the interaction records)
+  int collisionN = 0;
+  for (int collID = 0; collID < timesview.size(); ++collID) {
```

Comment:
Sorry @mfasDa I don't fully get what you mean here. I have of course included an enable/disable for the trigger configuration from the context inputs, but the first part is unclear to me. Do you mean that I should look for the L0 inputs in the same interaction record? I am asking because the condition in the code is for 12 BCs to issue a L0ack, so I was thinking of always running the trigger simulation, with only the global enable from context as possible veto for the simulation itself